doc/howtos/rdtraining/14_other_module.rst

```diff
@@ -0,0 +1,170 @@
+.. _howto/rdtraining/14_other_module:
+
+====================================
+Part 14: Interact With Other Modules
+====================================
+
+In the :ref:`previous chapter <howto/rdtraining/13_inheritance>`, we used inheritance to modify
+the behavior of a module. In our real estate scenario, we would like to go a step further
+and be able to generate invoices for our customers. Odoo provides an Invoicing module, so it
+would be neat to create an invoice directly from our real estate module: once a property
+is set to 'Sold', an invoice is created in the Invoicing application.
+
+Concrete Example: Account Move
+==============================
+
+.. note::
+
+    **Goal**: at the end of this section:
+
+    - A new module ``estate_account`` is created
+    - When a property is sold, an invoice is issued for the buyer
+
+    .. image:: 14_other_module/media/create_inv.gif
+        :align: center
+        :alt: Invoice creation
+
+Any time we interact with another module, we need to keep in mind the modularity. If we intend
+to sell our application to real estate agencies, some of them may want the invoicing feature but
+others may not want it.
+
+Link Module
+-----------
+
+The common approach for such use cases is to create a 'link' module. In our case, the module
```

Comment:
Doesn't really explain what a link module is e.g. 

```suggestion
The common approach for such use cases is to create a 'link' module: a module
which is automatically installed when other modules are selected and bridges
between them to provide *integration* between those otherwise unrelated
modules.

In our case, the module
```